_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,284 @@
+
+mvn -f ${workspace}/systemtest/pom.xml -P all verify -Dgroups=acceptance -DexcludedGroups=""
+```
+
+All available test groups are listed in the [Constants](https://github.com/strimzi/strimzi-kafka-operator/blob/master/systemtest/src/main/java/io/strimzi/systemtest/Constants.java)
+class. If you're interested, you can view our testing [document](https://github.com/strimzi/strimzi-kafka-operator/blob/master/development-docs/TESTING.md).
+
+## How system tests run in relation with Kubernetes
+
+As mentioned in the previous section, system tests run against a real Kubernetes cluster.
+You can run these tests against any Kubernetes cluster from `Minikube`, through to a big `Kubernetes` or `Openshift` cluster.
+In the `test` module we have a representation of a cluster that determines which environment it is running on.
+Moreover, we are using `Azure pipelines` to trigger these tests and using `Minikube`.
+
+## How to run them
+
+Running system tests is nothing to be scared of. In fact, it's easy. 
+
+The steps are as follows:
+
+1. Fork the [strimzi-kafka-operator](https://github.com/strimzi/strimzi-kafka-operator)
+2. Build the whole project in root directory - `mvn clean install -DskipTests`
+3. Set up your Kubernetes cluster
+4. Log in to your Kubernetes cluster
+5. Try to start some of our tests inside system the `test` module, for instance - `io.strimzi.systemtest.bridge.HttpBridgeST.testSendSimpleMessage()` (this takes around 5 minutes)
+
+It will run tests against the images as they are published in `Dockerhub` by default. 
+If you want to use your local images you should take a look on these environment variables:
+
+1. DOCKER_ORG = Specify the organization/repo containing the image used in system tests |`strimzi`|
+2. DOCKER_TAG = Specify the image tags used in system tests |`latest`|
+3. DOCKER_REGISTRY = Specify the docker registry used in system tests |`docker.io`|
+
+### Using custom images
+
+If you are interested in running custom images, you should take a look at the file that is located in the install directory: [here](https://github.com/strimzi/strimzi-kafka-operator/blob/master/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml).
+There are few attributes that you need to change:
+
+- `Cluster Operator` image = specify [here](https://github.com/strimzi/strimzi-kafka-operator/blob/master/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml#L26)
+- `Kafka` image = STRIMZI_KAFKA_IMAGES
+- `KafkaConnect` image  = STRIMZI_KAFKA_CONNECT_IMAGES
+- `KafkaConnectS2I` image =  STRIMZI_KAFKA_CONNECT_S2I_IMAGES
+- `KafkaMirrorMaker` image =  STRIMZI_KAFKA_MIRROR_MAKER_IMAGES
+- `KafkaMirrorMake2` image =  STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES
+- `Topic Operator` image =  STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+- `User Operator` image =  STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+- `KafkaBridge` image =  STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
+- `CruiseControl` image =  STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE
+
+And so on.
```

Comment:
Do we need this? It seems a bit odd here. Maybe change the sentence previous.: _There are few attributes that you need to change, including:_